build/templates/backend/scripts/arbitrum_archive.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+{{define "main" -}}
+
+set -e
+
+INSTALL_DIR={{.Env.BackendInstallPath}}/{{.Coin.Alias}}
+DATA_DIR={{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend
+
+NITRO_BIN=$INSTALL_DIR/nitro
+
+$NITRO_BIN \
```

Comment:
We have found that running this script demands write access to `/home/arbitrum` (seems like some data directory for the app?). It is not acceptable for us, is it possible to somehow override this with some option? We would like e.g. the `DATA_DIR` location

Reply:
Hmm... What is the setup you have? I am only seeing writes to the `DATA_DIR` locally. And the wasm error looks like something is wrong with the `INSTALL_DIR`. Should be using coin.alias for the directories, so I am surprised to see something like `/home/arbitrum`.